sync_test_suite/README.md

```diff
@@ -0,0 +1,206 @@
+
+![Time Error Bad](images/sync_G.8272_time-error-in-locked-mode_DPLL-to-PHC_PRTC-A_testimpl_bad.png)
+
+<figcaption class="figure-caption text-center">
+
+**Figure 5** Time Error Results above any PRTC mask. This indicates the TE is experiencing high fluctuations revealing very low accuracy.
+
+</figcaption>
+
+
+Now let's illustrate a case where the T-GM is reporting TE result within ITU-T recommendations. See below the TE distribution over a duration of 54000s. We can provide the following observations for the very same aspects commented in the Figure above:
+
+- Again the measured TE could vary over time. Because of this reason it is important to note that TE is measured over a sufficiently long time period. A minimum duration of 1000s under normal, locked operating conditions is recommended by [ITU G.8272](https://www.itu.int/rec/T-REC-G.8272/en). And this is one of the reasons why we let the user specify the duration of the measurement.
+
+- Again the measured TE can be negative or positive. A negative Time Error (TE) occurs when the measured clock signal under test arrives later than the reference clock signal. A positive Time Error (TE) occurs when the measured clock signal under test leads the reference clock signal.
+
+- The max|TE| in this case max|TE| reach 2ns, which is clearly inside the limits allowed by PRTC-A and PRTC-B classes.
+
+![Time Error Good](images/sync_G.8272_time-error-in-locked-mode_DPLL-to-PHC_PRTC-A_testimpl_good.png)
+
+<figcaption class="figure-caption text-center">
+
+**Figure 6** Time Error Results below PRTC-A (100ns) and PRTC-B (40ns) limits.
+
+</figcaption>
+
+
+
+### Understanding Maximum Time Interval Error
+
+First, let's define the time interval error (TIE). The TIE is the measure of the change in the TE over an observation interval, also referred to as τ(tau). Note that the TE itself is the phase error of the measured clock as compared to the reference clock. The change observed in the TE, or the TIE, is the difference between the observed TE values over an specific τ observation interval.
+
+The τ observation interval is a time interval that is generally fixed before the start of a measurement test. The TE is measured and recorded at the beginning of the test and after each observation interval time has passed; and the difference between the corresponding time error values gives the TIE for that observation interval.
+
+TIE measures the total error that a clock has accumulated as compared to a reference clock since the beginning of the test. Also, because TIE is calculated from TE, TIE is also measured using units of seconds. For any measurement, by convention the value of TIE is defined to be zero at the start of the measurement. If you observe a MTIE of zeros means that the TIE has not changed at all during the observation interval τ.
```

Comment:
Remove `For any measurement` from in front of
`by convention the value of TIE is defined to be zero at the start of the measurement.`